TelecomColourCode_Main.c

```diff
@@ -0,0 +1,79 @@
+/* ******************************************************************************
+ ***
+ ***     \file        TelecomColourCode_Main.c
+ ***     \ingroup     Session1-TCQ
+ ***     \author      Rohith K N
+ ***
+ ***     \brief       Main file to check different colour combinations
+ ***
+ ***     \par         File_description
+ ***                  This file explores more about code modularity
+ ***
+ *****************************************************************************/
+ 
+/*=============================================================================
+ =======                            INCLUDES                             =======
+ ============================================================================*/
+ /*------ standard includes -------*/
+#include <stdio.h>
+#include <assert.h>
+
+/*------ module includes --------*/
+#include "TelecomColourCode_Main.h"
+
+/*=============================================================================
+ =======               DEFINES & MACROS FOR GENERAL PURPOSE              =======
+ =============================================================================*/
+
+/*=============================================================================
+ =======                       CONSTANTS  &  TYPES                       =======
+ =============================================================================*/
+const char* MajorColorNames_a[] = {
+    "White", "Red", "Black", "Yellow", "Violet"
```

Comment:
The color enums and these texts are in different files. This makes it error-prone - a color may be added in one place and forgotten in another, leading to index-out-of-bounds. Any possibility to keep them together?